odk/extension/src/main/java/io/samagra/odk/collect/extension/handlers/ODKFormsHandler.kt

```diff
@@ -39,7 +46,12 @@ class ODKFormsHandler @Inject constructor(
     private val formsDatabaseInteractor: FormsDatabaseInteractor,
     private val storagePathProvider: StoragePathProvider,
     private val mediaUtils: MediaUtils,
-    private val entitiesRepository: EntitiesRepository
+    private val entitiesRepository: EntitiesRepository,
+    private var formsNetworkInteractor: FormsNetworkInteractor,
+    private var formsInteractor: FormsInteractor,
```

Comment:
You're accepting a `FormsInteractor`  inside the constructor of `ODKFormsHandler` which is itself an implementation of `FormsInteractor`.